citi-onboarding/colisio
# feature(nav-bar): Created the Navbar, missing the smooth scroll behavior.

What I did

- Created the Navbar implementing all it's funcionalities, except the smooth scroll behavior and the links.  
   - Logo on the left side of the screen, sections in the middle and button on the right side of the screen implemented, all with the hover transitions.

## How to test

1. Open the terminal and go to the folder of this project

2. In terminal:
   `git checkout develop`
   `git pull`
   `git checkout <branch name>`
   `git pull origin <branch name>`
   `yarn install`
   `yarn start`

1. In your browser go to http://localhost:port

1. Check if the component looks just like the mockup in different screen sizes 

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description
